client/state/ui/guided-tours/selectors.js

```diff
@@ -74,6 +78,12 @@ const getTourFromQuery = createSelector(
 	getInitialQueryArguments
 );
 
+export const isNewUser = state => {
+	const user = getCurrentUser( state );
+	const creation = Date.parse( user.date );
+	return ( Date.now() - creation ) <= moment.duration( 1, 'day' ).asMilliseconds();
```

Comment:
Wouldn't that lead to unintended results when someone signs up at 23:59 and reaches `/` a minute later?